docs/dev/cli/cli-commands.md

```diff
@@ -88,9 +87,16 @@ Options explanation:
 | `transformation download -c <csarname> -p <platformname>` or `transformation download -t <csarname/platformname>` | downloads the specific Transformation Artifact |
 | `transformation delete -c <csarname> -p <platformname>` or `transformation delete -t <csarname/platformname>` | deletes the specific Transformation from the Transformator |
 
+## Known Issues
+- some Exceptions after wrong  CLI inputs are not caught and displayed (options not supplied where it's needed)
+- setting inputs with spaces does not work
```

Comment:
Is this true even if quoted properly?  
E.g., instead of `input with spaces`, use `"input with spaces"`.
If the variant with propper quoting works, I would not say this is an issue - this is how most shells work.